### PR TITLE
PR Assignment1

### PR DESCRIPTION
**Pull request by dloriente**
Add Assignment1

### CODE REVIEW

**Comment by ocorcho** [comment]
Assignment 1 is ok.

File: Assignment4/dloriente-u12a065/task06.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  initNs = { "ns" : "http://somewhere#"}
+)
+for r in g.query(q64):
+  print(r.Pred, r.Obj)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+# TO DO
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.JohnSmith, vcard.work, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/dloriente-u12a065/task06.py:88]: vcard does not define the property "work". Define your own

File: Assignment4/dloriente-u12a065/task07.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    print(ss)
+for s,p,o in g.triples((None, RDF.type, ns.Person)):
+  print(s)
+
+# With SPARQL
+print("\n7.2 List all individuals of 'Person' with SPARQL:")
+q72 = prepareQuery('''
+  SELECT DISTINCT ?Sub ?Person
+  WHERE { 
+    {?Person RDFS:subClassOf ns:Person.
```
Comment (dgarijo) [inline Assignment4/dloriente-u12a065/task07.py:67]: use * to list the subclass hierarchy of Person.

File: Assignment4/dloriente-u12a065/task07.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+
+# With SPARQL
+print("\n7.3 List all individuals of 'Person' and all their properties including their class with SPARQL:")
+q73 = prepareQuery('''
+  CONSTRUCT {
+    ?Sub ?p ?o .
+  }
+  WHERE {
+    {?Person RDFS:subClassOf ns:Person.
```
Comment (dgarijo) [inline Assignment4/dloriente-u12a065/task07.py:110]: same as abbove

File: Assignment4/dloriente-u12a065/task07.py [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+for s,p,o in g73:
+  for ss,pp,oo in sorted(g.triples((s, None, None))):
+    print(ss,pp,oo)
+
+
+# With SPARQL
+print("\n7.3 List all individuals of 'Person' and all their properties including their class with SPARQL:")
+q73 = prepareQuery('''
+  CONSTRUCT {
+    ?Sub ?p ?o .
```
Comment (dgarijo) [inline Assignment4/dloriente-u12a065/task07.py:107]: you are asked to list the triples, not to create an rdf file.

**Review by dgarijo — CHANGES_REQUESTED** [review]

File: Assignment4/dloriente-u12a065/task06.py (hunk 1 above)
Comment (dloriente) [reply Assignment4/dloriente-u12a065/task06.py:88]: Done

File: Assignment4/dloriente-u12a065/task07.py (hunk 4 above)
Comment (dloriente) [reply Assignment4/dloriente-u12a065/task07.py:107]: Done

**Pull request merged**